cytosnake/utils/cyto_paths.py

```diff
@@ -139,20 +136,32 @@ def get_workflow_fpaths() -> dict:
     return file_search(workflow_path)
 
 
-def get_config_fpaths() -> dict:
-    """Obtains all file paths located in the `configs` folder as a dictionary.
+def get_config_dir_path() -> Path:
+    """Returns path to configuration folder
 
     Returns
     -------
-    dict
-        structured dictionary directory name and file paths as key value pairs
+    Path
+        Path to config directory
     """
+
     proj_root_path = get_project_root()
     config_path = proj_root_path / "configs"
     if not is_valid_path(config_path):
         raise FileNotFoundError("Unable to find config directory")
 
-    return file_search(config_path)
+    return config_path
+
+
+def get_config_fpaths() -> dict:
+    """Obtains all file paths located in the `configs` folder as a dictionary.
+
+    Returns
+    -------
+    dict
+        structured dictionary directory name and file paths as key value pairs
+    """
+    return file_search(get_config_dir_path())
```

Comment:
Could the code specified in the return be used more directly to avoid creating a new function? Doing this may simplify the code and reduce the complexity.

Reply:
agree that we want to reduce complexity! This is already quite complex!

I understand snakemake is particular about things, and this might be one of them, so please feel free to decide how to proceed.

Reply:
Hacky is ok sometimes!